libs/src/sdk/api/generated/default/apis/TipsApi.ts

```diff
@@ -15,40 +15,21 @@
 
 
 import * as runtime from '../runtime';
+import type {
+  GetTipsResponse,
+} from '../models';
 import {
-    GetTipsResponse,
     GetTipsResponseFromJSON,
     GetTipsResponseToJSON,
 } from '../models';
 
 export interface GetTipsRequest {
-    /**
-     * The number of items to skip. Useful for pagination (page number * limit)
```

Comment:
Huh. Really sad we're losing these... maybe I'll update the templates to include them again...